<launch>
  <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
    <param name="destination_frame" value="/base_footprint"/>
    <param name="cloud_destination_topic" value="/merged_cloud"/>
    <param name="scan_destination_topic" value="/scan_multi"/>
    <param name="laserscan_topics" value="/scan_filtered_right /scan_filtered_left" />
  </node>

  <node name="set_merge_params" pkg="dynamic_reconfigure" type="dynparam" args="set_from_parameters laserscan_multi_merger">
  	<param name="range_min" value="0.45"/>
   	<param name="range_max" value="10.1"/>
    <param name="angle_min" value="-3.14"/>
    <param name="angle_max" value="3.14"/>
  </node>

  <node pkg="laser_filters" type="scan_to_scan_filter_chain" name="laser_filter_scan_multi">
    <remap from="scan" to="/scan_multi" />
    <remap from="scan_filtered" to="/scan_multi_angle_filter"/>
    <rosparam command="load" file="$(find wheelchair_description)/params/angle_scan_multi_filter.yaml" />
  </node>

</launch>
